### DISCUSSION TITLE
How do I share the notion database with the integration I have created?

### DISCUSSION THREAD

**Discussion by egeres** [lastorel/pytion#59] [Q&A] [answered]
It seems that it used to be possible to share the notion database by clicking "share" and then "invite" to then select your integration. Seeing [this video](https://www.youtube.com/watch?v=Hk7Vk_v4yfo) I see a listed option to share the notion database with an integration was available before, but I can't find it nowadays. Am I missing something?

![image](https://user-images.githubusercontent.com/8045792/211676411-b0a690e9-e2bd-4207-9e7b-0ee29f21ebe5.png)

(I'm stuck here because I'm getting the error `The resource does not exist or the resource has not been shared with owner of the bearer token.0` when calling `no.databases.get`, but it seems to me the root of the problem is my integration not having permissions to see/edit the DB, otherwise, let me know!)

**Comment by lastorel** [comment, marked as answer]
Add your new integration 'manager' to your pages or databases by going to the page's menu (three dots), at the bottom of the menu clicking on Add connections and then searching for you new Intigration. (c)

**Comment by egeres** [comment]
🙃 I missed the name of my integration while scrolling (many times) and was confused by the documentation regarding this, thank you so much for the help and all the work you're putting into the library!!
